ordersPage.js

```diff
@@ -79,6 +80,12 @@ export function addOrders(orderData)
     ticketsInput.addEventListener("change",() => {
       orderChangeHandler(ticketCategorySelect,ticketsInput,updateButton);
     });
+
+    deleteButton.addEventListener("click", () => {
+      DeleteOrder(order.orderID).then(data => console.log("deleted",data));
+      ordersContainer.removeChild(orderCard);
```

Comment:
You need to call `removeChild` function in then block.

Reply:
Oops, mi-a scapat. Modific imediat.